packages/components/pie-checkbox/src/index.ts

```diff
@@ -125,11 +137,11 @@ export class PieCheckbox extends FormControlMixin(RtlMixin(LitElement)) implemen
             <input
                 type="checkbox"
                 .value=${value}
-                ?checked=${live(checked)}
+                .checked=${!!checked}
```

Comment:
The `checked` _attribute_ (set in Lit using `?checked`) refers to the initial state for the checkbox ([MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox#checked)), whereas the `checked` _property_ (the same as `document.querySelector('input[type="checkbox"]').checked`, set using `.checked` controls the actual checked state.

Took me a while to work that one out!